dockstore-integration-testing/src/test/java/io/dockstore/client/cli/GeneralIT.java

```diff
@@ -1313,6 +1313,29 @@ public void testUpdateToolPathDockerfile() throws ApiException {
         assertEquals("the cwl path should be changed to /test1/Dockerfile", "/test1/Dockerfile", path);
     }
 
+    /**
+     * Test to update the tool's forum and it should change the in the database
+     *
+     * @throws ApiException
+     */
+    @Test
+    public void testUpdateToolForumUrl() throws ApiException {
+        //setup webservice and get tool api
+        ContainersApi toolsApi = setupWebService();
+
+        DockstoreTool toolTest = toolsApi.getContainerByToolPath(DOCKERHUB_TOOL_PATH, null);
+        toolsApi.refresh(toolTest.getId());
+
+        //change the forumurl
+        toolTest.setForumUrl("hello.com");
+        toolsApi.updateContainer(toolTest.getId(), toolTest);
+        toolsApi.refresh(toolTest.getId());
+
+        //check the tool's forumurl is updated in the database
+        final String updatedForumUrl = testingPostgres.runSelectStatement("select forumurl from tool where id = " + toolTest.getId(), String.class);
+        assertEquals("the forumurl should be hello.com", "hello.com", updatedForumUrl);
```

Comment:
can use constant here and on line 1330 to make it really obvious what's going on